src/bindings.cpp

```diff
@@ -146,7 +146,8 @@ namespace madrona_gpudrive
             })
             .def("deleted_agents_tensor", &Manager::deletedAgentsTensor)
             .def("map_name_tensor", &Manager::mapNameTensor)
-            .def("scenario_id_tensor", &Manager::scenarioIdTensor);
+            .def("scenario_id_tensor", &Manager::scenarioIdTensor)
+            .def("episode_length_tensor", &Manager::episodeLengthTensor);
```

Comment:
We already have the steps remaining tensor, is the only diff that this one is counting up?